driver/example_test.go

```diff
@@ -40,14 +40,14 @@ func ExampleDriver() {
 
 	_ = it.Err()
 
-	_, _ = c.Delete(ctx, "db1", "c1", Filter(`{"F1":"V1"}`), &DeleteOptions{})
+	_, _ = c.Delete(ctx, "db1", "c1", Filter(`{"F1":"V1"}`))
 
 	tx, _ := c.BeginTx(ctx, "db1", nil)
 	defer func() { _ = tx.Rollback(ctx) }()
 
-	_, _ = tx.Insert(ctx, "c1", []Document{Document(`{"F1":"V1"}`)}, &InsertOptions{})
+	_, _ = tx.Insert(ctx, "c1", []Document{Document(`{"F1":"V1"}`)})
 
-	it, _ = tx.Read(ctx, "c1", Filter(`{"F1":"V1"}`), &ReadOptions{})
+	it, _ = tx.Read(ctx, "c1", Filter(`{"F1":"V1"}`), Fields("{}"))
```

Comment:
Same as above.

Reply:
Is there a way to not have to specify it if its empty?

Reply:
There is no clean way to do this. Cleanest is probably to add `ReadWithFields()`
But because this is an integration API there is no use case for read all fields call.

For example in CLI I will use it like this:
`Read(filter, fields)`

Not like this:
if fields != nil && fields != `{}` {
  err = ReadWithFields(filter, fields)
 } else {
   err = Read(filter)
 }

Reply:
`Read(filter, fields)` just call this always?